templates/containerized-sdk/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  emlinux3-sdk:
+    image: emlinux3-sdk
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        uid: ${UID:-1000}
+        http_proxy: $http_proxy
+        https_proxy: $https_proxy
+        ftp_proxy: $ftp_proxy
+        no_proxy: $no_proxy
+    environment:
+      - http_proxy=$http_proxy
+      - https_proxy=$https_proxy
+      - ftp_proxy=$ftp_proxy
+      - no_proxy=$no_proxy
+    volumes:
+      - /proc:/proc:rw
+      - /dev/shm:/dev/shm:rw
```

Comment:
Can we mount the host OS's home directory to share files between the host OS and the container?

e.g.
```
      - /home:/home-host:rw
```

Reply:
Added  following line to share user's $HOME with container.

```
- $HOME:/host-home:rw
```

Reply:
Thanks!